pkg/proxy/apis/config/validation/validation_test.go

```diff
@@ -616,7 +616,7 @@ func TestValidateKubeProxyConntrackConfiguration(t *testing.T) {
 				TCPEstablishedTimeout: &metav1.Duration{Duration: 5 * time.Second},
 				TCPCloseWaitTimeout:   &metav1.Duration{Duration: -5 * time.Second},
 			},
-			expectedErrs: field.ErrorList{field.Invalid(newPath.Child("KubeConntrackConfiguration.TCPCloseWaitTimeout"), metav1.Duration{Duration: -5 * time.Second}, "must be greater than or equal to 0")},
+			expectedErrs: field.ErrorList{field.Invalid(newPath.Child("KubeConntrackConfiguration.TCPCloseWaitTimeout"), &metav1.Duration{Duration: -5 * time.Second}, "must be greater than or equal to 0")},
```

Comment:
Why did this change?

Reply:
It is a string formatting difference. Since the `TCPCloseWaitTimeout` above is a pointer to a `metav1.Duration`, then this not being a pointer creates a differently formatted string.